CMakeLists.txt

```diff
@@ -371,7 +371,9 @@ if(BUILD_TEST)
     set_property(SOURCE ${JIT_TEST_SRCS} APPEND PROPERTY COMPILE_OPTIONS "-Werror")
   endif()
 
-  install(TARGETS ${NVFUSER_TESTS} DESTINATION bin)
+  if(BUILD_INSTALL)
```

Comment:
definitely not a big fan of this! Let's fix CMAKE_INSTALL_PREFIX for our standalone build.

Let's back these out and do that in follow up PRs.